Makefile

```diff
@@ -1,5 +1,5 @@
 __SIM_ID=`xcrun simctl list|egrep -m 1 '$(SIM_NAME) \([^(]*\) \([^(]*\)$$'|sed -e 's/.* (\(.*\)) (.*)/\1/'`
```

Comment:
For fear of breaking something, i'm going to leave it.

Essentially, it's just grabbing a simulator ID, and the ID's are weirdly formatted UUID's.